libs/sdk-ui-kit/src/utils/tests/immutable.test.ts

```diff
@@ -0,0 +1,52 @@
+// (C) 2007-2020 GoodData Corporation
+import { fromJS } from "immutable";
+
+import { propsEqual } from "../immutable";
+
+describe("immutable", () => {
+    let propsA: any;
+    let propsB: any;
+    let propsC: any;
+    let propsD: any;
+    let propsE: any;
+    let propsF: any;
+
+    beforeEach(() => {
+        propsA = {
+            prop1: { prop: true },
+            prop2: [{ prop3: "abc" }],
+        };
+
+        propsB = {
+            prop1: { prop: true },
+            prop2: [{ prop3: "abc" }],
+        };
+
+        propsC = {
+            prop1: { prop: false },
+            prop2: [{ prop4: "def" }],
+        };
+
+        const prop1 = fromJS({ prop: true });
+        const prop2 = fromJS([{ prop: "abc" }]);
+
+        propsD = { prop1, prop2 };
+        propsE = { prop1, prop2 };
+        propsF = {
+            prop1: fromJS({ prop: true }),
+            prop2: fromJS([{ prop: "abc" }]),
+        };
+    });
+
+    describe("propsEqual", () => {
+        it("should deep-compare non-immutable props", () => {
```

Comment:
These should be split to parametric tests (same with the other `it`).

Reply:
Fixed.